### PR TITLE
Event queues with localstack implementation

### PR DESCRIPTION
**Pull request by ejjick**

### CODE REVIEW

File: web/kyc/tasks.py [hunk 1]
```diff
@@ -0,0 +1,146 @@
...
+    netverify_client = get_netverify_client()
+    callback = NetverifyCallback.objects.get(id=callback_id)
+    # lookup scan so can get user who we need to send email to
+    scan = netverify_client.get_scan_with_jumio_id_scan_reference(
+        callback.jumio_id_scan_reference
+    )
+    send_email = False
+    if scan is None:
+        callback.processed_status = NetverifyCallback.PROCESSED_STATUS_FAILED # noqa
+        # TODO - clean up this duplication.
```
Comment (cballantyne) [inline web/kyc/tasks.py]: Not great. But I don't want to complicate this PR too much so have put a TODO for this.

**Comment by cballantyne** [comment]
I think this is ready to review. 
I have left out anything to do with running this in staging/production - this will come in separate PR once this one is merged in.

File: web/kyc/tasks.py (hunk 1 above)
Comment (ejjick) [reply web/kyc/tasks.py]: Might be able to assign `output = logger` to control the output

Comment (cballantyne) [reply web/kyc/tasks.py]: I have fixed this up so tests don't depend on stdout. Those stdout writes now just go to logger. And the tests validate on what happened inside the method. I mock the send email methods to determine if they were called or not.

**Pull request merged**